x-pack/plugins/security_solution/tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "scripts/**/*",
     // have to declare *.json explicitly due to https://github.com/microsoft/TypeScript/issues/25636
     "server/**/*.json",
+    "scripts/**/*.json",
```

Comment:
This is redundant now.

Reply:
I did a clean, boot then ran ES/kibana without this and was able to run your script and create the roles. 😅 Consider trying that on your end and see if it works too